ReleaseProcessAutomation/ReleaseProcessAutomation.csproj

```diff
@@ -1,17 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <PackageId>ReleaseProcessAutomation</PackageId>
-        <Version>1.0.0</Version>
-        <Authors>re-motion</Authors>
-        <Company>RUBICON</Company>
         <OutputType>Exe</OutputType>
         <TargetFramework>net6.0</TargetFramework>
         <PackAsTool>true</PackAsTool>
         <ToolCommandName>rpa</ToolCommandName>
         <Nullable>enable</Nullable>
         <RootNamespace>ReleaseProcessAutomation</RootNamespace>
     </PropertyGroup>
+    
+    <PropertyGroup>
+        <PackageId>Remotion.ReleaseProcessAutomation</PackageId>
+        <Version>1.0.0</Version>
+        <Authors>rubicon IT GmbH</Authors>
+        <Owners>rubicon IT GmbH</Owners>
```

Comment:
According to the doc there is no `Owners` property, it does not seem to exist with the pack target. I checked the re-motion packages and there the owner is also empty. So I think we can remove this property.